civictechprojects/static/css/_vars.scss

```diff
@@ -107,7 +92,26 @@ $color-background-light: $color-neutral-10; //individual components
 // #####
 
 
-//note: everything below here should be checked and brought into DS, these are live values
+//note: everything below here should be checked and brought into DS, these are live values!
+
+//## base colors (NOT design system implementation yet; phase these out with e.g. $color-orange -> $color-brand-70) 
+$color-orange: #f79e02; // brand
+$color-orange-dark: #de8e01; // accent/button borders
+$color-orange-light: #f7cf88; // footer button hover
+$color-orange-text: #f06d23; // companies page, will need a rename
```

Comment:
for any text that needs to be set in orange, apply $color-brand-80 | #F06D23 //$color-text-decorative
Text color for names and emphasized titles or subtitles on the content page.  *This color is created for accessibility purposes to meet color contrast requirements

Reply:
Got it, good catch. I'll update.

Reply:
Also, it should apply to "read more/read less" text that expands the positions available on the Project Profile page. :)